server/src/types/MediaPlayerResponse.ts

```diff
@@ -0,0 +1,62 @@
+export interface MediaPlayerResponse {
+	state: string;
```

Comment:
do mind typing this like `'buffer' | 'pause' | 'startup' | 'playing'` to help with intellisense some more. Looks good otherwise

Reply:
No problem. Changed this in 840136c